draft-ietf-ppm-dap.md

```diff
@@ -3961,7 +3965,7 @@ fact that some measurement is being taken at all.
 
 Such enumeration attacks can be mitigated by incorporating unpredictable values
 into the task ID derivation. They do not, however, affect the protocol goals of
-privacy or robustness.
+privacy or verifiability.
```

Comment:
"They do not, however, affect the core security goals of VDAFs ({{Section 9 of !VDAF}})."